src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -70,7 +70,6 @@ public static IServiceCollection AddApplicationInsights(this IServiceCollection
             });
 
             services.AddSingleton<ITelemetryInitializer, WebJobsRoleEnvironmentTelemetryInitializer>();
-            services.AddSingleton<ITelemetryInitializer, WebJobsSanitizingInitializer>();
```

Comment:
Similar to performance work I did in the Functions Host system logging, we don't want to be running sanitizer on every single piece of data. Instead, we should run any required sanitization at the call site for any data we think might be sensitive.